installation/linux.md

```diff
@@ -625,6 +613,42 @@ sudo chown -hR openhab:openhab /opt/openhab2
 sudo systemctl start openhab2.service
 ```
 
+#### Upgrade
+
+To stay up to date with new releases, you should do regular upgrades of your manual installation.
+This is especially important if you are working with the latest snapshot as changes and fixes are incorporated [constantly](https://openhab.jfrog.io/openhab/webapp/#/builds/openHAB-Distribution).
+
+openHAB uses a script to update to any other version, or from stable to snapshot and visa-versa.
+Your personal configuration will be retained on upgrades, but you should **stop openHAB** and perform a backup first.
+
+* **Versions 2.1.0 and newer**
+
+  From versions 2.1.0 above, openHAB is distributed with the update script included. 
+  This script should be called from within openHAB's root directory. 
+  Assuming the openHAB directory is in `/opt/openhab`, simply run the following commands to update to the next major version of openHAB:
+
+  ```bash
+  cd /opt/openhab
+  sudo runtime/bin/update
+  ```
+
+  You may also change to openHAB's more frequent, but less stable snapshot builds. 
+  Just append "-SNAPSHOT" to the target version, e.g.:
+
+  ```bash
+  sudo runtime/bin/update 2.2.0-SNAPSHOT
+  ```
+
+* **Older Versions**
+
+  If you're using a version earlier than 2.1.0, then you can use the following commands in Terminal to download the script and run it. 
+  Assuming the openHAB directory is in `/opt/openhab`:
+
+  ```bash
+  cd /opt/openhab
+  sudo sh -c "$(curl -fsSL https://raw.githubusercontent.com/openhab/openhab-distro/master/distributions/openhab/src/main/resources/bin/update)"
```

Comment:
-c follows a slightly stranger syntax:

```bash
sh -c [command] -- [param 1] [param 2]
```

```bash
sudo sh -c "$(curl -fsSL https://raw.githubusercontent.com/openhab/openhab-distro/master/distributions/openhab/src/main/resources/bin/update)" -- 2.1.0
```

Do you think that we should put this specifically?